backend/src/main/java/de/neuefische/backend/api/dto/ApiResponseAddress.java

```diff
@@ -0,0 +1,11 @@
+package de.neuefische.backend.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record ApiResponseAddress(
+        @JsonProperty("strasse")
+        String streetAndHomeNumber,
+        @JsonProperty("ortStrasse")
```

Comment:
"strasse" and "ortStrasse" i don't understand the logic...?

Reply:
This naming comes from external ArbeitsagenturApi, I can´t rename it.